esmvalcore/_recipe.py

```diff
@@ -445,7 +476,10 @@ def _get_input_files(variable, config_user):
                 '\n'.join(input_files))
     if (not config_user.get('skip-nonexistent')
             or variable['dataset'] == variable.get('reference_dataset')):
-        check.data_availability(input_files, variable)
+        if variable['frequency'] != 'fx':
```

Comment:
@mattiarighi @bouweandela so shall we put back in the checks for fx files availability?

Reply:
Yes, I would put it back.

Reply:
it is le put back :grin: